modules/099-ceph-csi/enabled

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Copyright 2021 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+source /deckhouse/shell_lib.sh
+
+function __main__() {
+  enabled::disable_module_if_cluster_is_not_bootstraped
```

Comment:
We may want to use the CSI driver even if a cluster is not bootstrapped, e.g., deploying a network gateway.

Reply:
I didn't find any reason why we couldn't do that. Removed the if_cluster_is_not_bootstraped check.